Blockzilla/OverlayView.swift

```diff
@@ -171,8 +171,7 @@ class OverlayView: UIView {
      - Returns: An NSAttributedString with `phrase` localized and styled appropriately.
      
      */
-	func getAttributedButtonTitle(phrase: String,
-								  localizedStringFormat: String) -> NSAttributedString {
+	func getAttributedButtonTitle(phrase: String, localizedStringFormat: String) -> NSAttributedString {
```

Comment:
This was a swiftlint error that somehow got merged into master. Had to fix it to commit.